cmd/serve.go

```diff
@@ -110,6 +110,9 @@ func init() {
 
 	RootCmd.Flags().DurationVar(&tokenTtl, "token-ttl", 24*time.Hour, "TTL for the token")
 
+	RootCmd.Flags().StringVar(&keypairFile, "keypair-file", "signing", `Path to private key used for token signing.
```

Comment:
--signer-private-key (mandatory)
--signer-public-key (optional, defaults to $(signer-private-key).pub)

Reply:
> mandatory

so, no default then?

> --signer-public-key (optional, defaults to $(signer-private-key).pub)

I do dislike too many configuration options. What's the use case for changing it? There is no easy/standard way outside of this program to generate these files, those are just opaque binary data, why would anyone put them in specific files?

It would be nicer if we could just put all the signing stuff into one file, or alternatively, provide an outside way to generate this keypair(like ssh-keygen, or some short openssl command), which might imply file names to use.

But it is slowly getting into "too many words, too little code" area, you might want to just code it down your way yourself as a fork maintainer.